__tests__/app.test.js

```diff
@@ -69,6 +69,22 @@ describe("GET /api/articles/:article_id", () => {
       })
     })
   })
+  test("200: Responds with comment count for articles with article id", () => {
+    return request(app)
+    .get('/api/articles/1?comment_count=true')
+    .expect(200)
+    .then((response) => {
+        expect(response.body.comment_count).toBe(11);
+    });
+  });
+  test("400: Sends an appropriate status and error message when given an comment query", () => {
+    return request(app)
+      .get('/api/articles/3/?comment_count=hi')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Invalid comment count query');
+      });
```

Comment:
You have interpreted the task in a different way potentially here, we're looking to add a new key to the returned object on the GET request, we don't need to pass in `comment_count` as a query. This would be a good approach if we did want `comment_count` to be a query though!